fastlane_bot/helpers/poolandtokens.py

```diff
@@ -403,10 +403,25 @@ def _carbon_to_cpc(self) -> ConstantProductCurve:
         allow to omit yint (in which case it is set to y, but this does not make
         a difference for the result)
         """
+        def calculate_parameters(y: Decimal, pa: Decimal, pb: Decimal, pm: Decimal, n: Decimal):
```

Comment:
missing tests for this functionality - check_overlap and "cleanup" of overlapping strategies